### ISSUE TITLE
How to enable readConcern `Majority`

### ISSUE THREAD

**Issue by xromp** [nodkz/mongodb-memory-server#160]
I'm setting up a unit testing using mongodb-memory -server and in my development project I have ```mongo watch``` and it listens to one of my collection. When I run the unit test, it throws an error: 

> Error [ERR_UNHANDLED_ERROR]: Unhandled error. (MongoError: Storage engine does not support read concern: { readConcern: { level: "majority" } })

But it works when running using the live mongdb.

Thanks,
Rom

**Comment by nodkz** [comment]
You need to try other `storageEngine`  option. See https://github.com/nodkz/mongodb-memory-server#available-options

Or maybe need to use ReplicaSet.

Anyway, storage by default `ephemeralForTest ` does not support readConcern.

**Comment by hasezoey** [comment]
question solved, closing

> MongoError: Storage engine does not support read concern: { readConcern: { level: "majority" } }

>  storage by default ephemeralForTest does not support readConcern.

**Comment by hasezoey** [comment]
sorry, i could have probably worded that last comment better, i basically just noted that this issue had been solved by a previous comment https://github.com/nodkz/mongodb-memory-server/issues/160#issuecomment-476981949 and included the reasons for why it was solved, i should have probably included a fix too.

so let me re-iterate and with the current state of MMS 9.0:

Error `MongoError: Storage engine does not support read concern: { readConcern: { level: "majority" } }` will show up because storage-engine `ephemeralForTest`, which is the default for versions before mongodb 7.0, does not support read-concerncs. The fix is to change the storage engine to a different engine like `wiredTiger` (which in 5.0 and 6.0 is the only other options).
Also to note is that mongodb 7.0 removed storage engine `ephemeralForTest` and so `wiredTiger` is the only one left there.

TL;DR:
use mongodb version 7.0 OR `(new )MongoMemoryServer(.create)({ instance: { storageEngine: 'wiredTiger' } })` or `(new )MongoReplSet(.create)({ replSet: { storageEngine 'wiredTiger' } })`.
